src/werkzeug/local.py

```diff
@@ -5,11 +5,22 @@
 
 # Each thread has its own greenlet, use that as the identifier for the
 # context. If greenlets are not available fall back to the current
-# thread ident.
+# thread ident. Asyncio Task ident is preferred.
```

Comment:
Asyncio is now "preferred", but is only given a short sentence at the *end* of this explanation comment.